exhibitor-standalone/src/main/java/com/netflix/exhibitor/application/ExhibitorMain.java

```diff
@@ -124,7 +126,16 @@ public ExhibitorMain(BackupProvider backupProvider, ConfigProvider configProvide
 
         DefaultResourceConfig   application = JerseySupport.newApplicationConfig(new UIContext(exhibitor));
         ServletContainer        container = new ServletContainer(application);
-        server = new Server(httpPort);
+
+        SelectChannelConnector connector = new SelectChannelConnector();
```

Comment:
Using the NIO connector is probably the important change -- the other options can probably be omitted, or set to sane values to prevent runaway floods from making Exhibitor hit any EMFILEs.